packages/qdrant-loader-core/src/qdrant_loader_core/graph/schema/__init__schema.py

```
@@ -0,0 +1,56 @@
+        "CREATE INDEX ON :Container(id)",
+        "CREATE INDEX ON :Label(id)",
+        "CREATE INDEX ON :Concept(id)",
+        "CREATE INDEX ON :Chunk(id)",
+    ]
+
+    for q in queries:
+        try:
+            await graph_store.query_cypher(q, {})
+        except Exception:
+            # already exists → idempotent
+            pass
```

Ruff (0.15.14)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)